src/Vulkan/Silk.NET.Vulkan/Vk.cs

```diff
@@ -221,23 +300,33 @@ public unsafe bool IsDeviceExtensionPresent(PhysicalDevice device, string extens
         /// <returns>Whether the device extension is available.</returns>
         public unsafe bool IsDeviceExtensionPresent(Instance instance, string extension, out PhysicalDevice device)
         {
-            var physicalDeviceCount = 0u;
-            EnumeratePhysicalDevices(instance, &physicalDeviceCount, null);
-            var physicalDevices = stackalloc PhysicalDevice[(int) physicalDeviceCount];
-            EnumeratePhysicalDevices(instance, &physicalDeviceCount, physicalDevices);
+            device = GetPhysicalDevices(instance).FirstOrDefault(pd => IsDeviceExtensionPresent(pd, extension));
+            return device.Handle != IntPtr.Zero;
```

Comment:
```suggestion
            return GetPhysicalDevices(instance).Any(pd => IsDeviceExtensionPresent(pd, extension));
```

Reply:
Your suggestion would not initialise `device` correctly, which is an `out` parameter.

Reply:
LGTM

Reply:
Approved by @Perksey